src/main/scala/io/delta/sources/package.scala

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (2020) The Delta Lake Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.delta
+
+import org.apache.spark.sql.{DataFrame, DataFrameReader, DataFrameWriter}
+import org.apache.spark.sql.streaming.{DataStreamReader, DataStreamWriter, StreamingQuery}
+
+package object sources {
```

Comment:
In a second thought, I think `import io.delta.implicits._` as this is similar to `import spark.implicits._` which people are familiar.

Could you add a `package object implicits` in the `src/main/scala/io/delta/package.scala` file?

Reply:
I think it's a good idea. I have renamed the package to implicits.